lib/sentry/interfaces.ex

```diff
@@ -267,4 +267,62 @@ defmodule Sentry.Interfaces do
       :stacktrace
     ]
   end
+
+  defmodule Span do
+    @moduledoc """
+    The struct for a tracing **span**.
+
+    See <https://develop.sentry.dev/sdk/event-payloads/span>.
+    """
+
+    @moduledoc since: "11.0.0"
+
+    alias Sentry.Interfaces
+
+    @typedoc since: "11.0.0"
+    @type t() :: %__MODULE__{
+            start_timestamp: String.t() | number(),
+            timestamp: String.t() | number(),
+            span_id: String.t(),
+            parent_span_id: String.t() | nil,
+            description: String.t(),
+            tags: map(),
+            trace_id: String.t() | nil,
+            op: String.t(),
+            contexts: Interfaces.context(),
```

Comment:
Spans do not have contexts themselves in the protocol, in some SDKs this is held on the `Transaction` object as a reference and then merged into the top level `contexts` on the event payload.
It depends on implementation if you want to also have this reference here or not.